### PR TITLE
Add Python and Ruby docs

### PR DESCRIPTION
**Pull request by jirikuncar**

### CODE REVIEW

File: docs/ruby.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+{% highlight ruby %}
+require 'cucumber'
+require 'ddtrace'
+
+## Configure default Cucumber integration
+Datadog.configure do |c|
+  c.use :cucumber, options
+end
+
+## Example of how to attach tags from scenario to active span
```
Comment (juan-fernandez) [inline docs/ruby.md:28]: is this all expected to be within the syntax highlight?

**Review by juan-fernandez — APPROVED** [review]
🚀

File: docs/ruby.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+# Cucumber Instrumentation
+
+Cucumber integration will trace all executions of scenarios and steps when using `cucumber` framework.
+
+To activate your integration, use the `Datadog.configure` method:
+
+{% highlight ruby %}
+require 'cucumber'
+require 'ddtrace'
```
Comment (jirikuncar) [inline docs/ruby.md:21]:
```suggestion
require 'ddtrace'
{% endhighlight %}
```

File: docs/ruby.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+## Configure default Cucumber integration
+Datadog.configure do |c|
+  c.use :cucumber, options
+end
```
Comment (jirikuncar) [inline docs/ruby.md:26]:
```suggestion

{% highlight ruby %}
Datadog.configure do |c|
  c.use :cucumber, options
end
{% endhighlight %}
```

File: docs/ruby.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+## Example of how to attach tags from scenario to active span
+Around do |scenario, block|
```
Comment (jirikuncar) [inline docs/ruby.md:29]:
```suggestion
{% highlight ruby %}
Around do |scenario, block|
```

File: docs/ruby.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+Cucumber integration will trace all executions of scenarios and steps when using `cucumber` framework.
+
+To activate your integration, use the `Datadog.configure` method:
+
+{% highlight ruby %}
+require 'cucumber'
+require 'ddtrace'
+
+## Configure default Cucumber integration
```
Comment (jirikuncar) [inline docs/ruby.md:23]:
```suggestion
# Configure default Cucumber integration
```

File: docs/ruby.md (hunk 1 above)
Comment (jirikuncar) [inline docs/ruby.md:28]:
```suggestion
# Example of how to attach tags from scenario to active span
```

Comment (jirikuncar) [reply docs/ruby.md:28]: yes

File: docs/ruby.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+RSpec integration will trace all executions of example groups and examples when using `rspec` test framework.
+
+To activate your integration, use the `Datadog.configure` method:
+
+{% highlight ruby %}
+require 'rspec'
+require 'ddtrace'
+
+## Configure default RSpec integration
```
Comment (jirikuncar) [inline docs/ruby.md:67]:
```suggestion
# Configure default RSpec integration
```

File: docs/ruby.md (hunk 1 above)
Comment (jirikuncar) [reply docs/ruby.md]: it's a copy&paste from Ruby docs

**Pull request merged**